test/evp_test.c

```diff
@@ -1039,10 +1039,12 @@ static int mac_test_init(EVP_TEST *t, const char *alg)
             && strcmp(alg + sz - (sizeof(epilogue) - 1), epilogue) == 0)
             sz -= sizeof(epilogue) - 1;
 
+        if (strncmp(alg, "CMAC", sz) == 0) {
+            t->skip = 1;
+            return 1;
+        }
```

Comment:
I see no reason to skip CMAC by EVP_PKEY unconditionally...  or should we stop doing *any* of the MAC by EVP_PKEY tests?

Reply:
@t8m said to just skip the tests because CMAC has its own tests.  I can make that change based on no-deprecated if you prefer, but you and Tomas need to come to agreement.

Reply:
Found the review comment with that conversation...

Reply:
Marking this thread as resolved, discussion continuing in the thread at  https://github.com/openssl/openssl/pull/13829#discussion_r554947635